test/runtime_tests.cc

```diff
@@ -1004,5 +1004,65 @@ namespace detail {
 		dry_run_with_nodes(nodes);
 	}
 
+	TEST_CASE_METHOD(test_utils::runtime_fixture, "kernels can access 0-dimensional buffers", "[buffer]") {
+		constexpr float value_a = 13.37f;
+		constexpr float value_b = 42.0f;
+
+		buffer<float, 0> buf_0 = value_a;
```

Comment:
That's an extension, right? Pretty cool!

Reply:
I think so - but legitimate IMO since 0-dimensional buffers are just "managed variables". In that spirit, it's semantically somewhat unfortunate that SYCL defaults `Dims` to 1.